src/plugins/events.go

```diff
@@ -254,9 +254,13 @@ func (a *Events) sendConfigApplyEvent(msg *core.Message) {
 			command.Meta.GetMessageId(),
 		)
 	} else if nginxConfigResponse.Status.Status == proto.CommandStatusResponse_CMD_ERROR {
+		nginxId := ""
+		if nginxConfigResponse.GetConfigData() != nil {
+			nginxId = nginxConfigResponse.GetConfigData().NginxId
```

Comment:
.GetNginxId()

we should use the functions rather than the properties directly